indico/web/client/js/jquery/widgets/jinja/permissions_widget.js

```diff
@@ -193,6 +194,7 @@ import Palette from 'indico/utils/palette';
       const self = this;
       return $('<button>', {
         'type': 'button',
+        'title': $T.gettext("Delete entry '{0}'".format(principal.name || principal.id)),
```

Comment:
Formatting needs to happen outside the gettext call:

```suggestion
        'title': $T.gettext("Delete entry '{0}'").format(principal.name || principal.id),
```